plugins/LayerView/LayerView.qml

```diff
@@ -127,6 +127,16 @@ Item
                     } else {
                         view_settings.show_legend = false;
                     }
+
+                    typesLegenModelNoCheck.clear();
+                    typesLegenModelNoCheck.append({
+                        label: catalog.i18nc("@label", "Top / Bottom"),
+                        colorId:  "layerview_skin"
+                    });
+                    typesLegenModelNoCheck.append({
+                        label: catalog.i18nc("@label", "Inner Wall"),
+                        colorId:  "layerview_inset_x"
+                    });
```

Comment:
Why is this necessary? The `typesLegenModelNoCheck` is only initialised with these two settings, and never changed by anything afterwards. Clearing and re-appending its items should do nothing, right?